app/addons/components/react-components.react.jsx

```diff
@@ -1081,6 +1105,114 @@ function (app, FauxtonAPI, React, Components, ace, beautifyHelper) {
 
   });
 
+  var ToggleState = React.createClass({
+    getDefaultProps: function () {
+      return {
+        defaultLeft: 'true'
+      };
+    },
+    render: function () {
+      var config = this.props.toggleConfig,
+          defaultLeft = false,
+          defaultRight = false;
+
+      var title = config.title,
+          leftLabel = config.leftLabel,
+          rightLabel = config.rightLabel,
+          leftClick = config.leftClick,
+          rightClick = config.rightClick,
+          enclosingID = config.enclosingID;
+
+      if (config.defaultLeft) {
+        defaultLeft = "checked";
+        defaultRight = false;
```

Comment:
Can these just be true/false? Seems a little odd to have the values be both strings and booleans. I think "true" works.